src/models/user.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class user extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */ 
+    static associate(models) {
+      // define association here
+    }
+  };
+  user.init({
+    firstName: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true,
+      },
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true,
+      },
+    },
+    email: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true,
+      },
+    },
+    password: {
+      type: DataTypes.STRING,
+      validate: {
+        notEmpty: true,
+      },
+    }
+  }, {
+    sequelize,
+    modelName: 'user',
+  });
+  return user;
+};
```

Comment:
Falta línea en blanco al final (el linter suele detectar estos casos).

```suggestion
};

```